### ISSUE TITLE
Default Post Functionality: Set post page

### ISSUE THREAD

**Issue by rcass** [redacademy/it-winter-2018#14]
Under the reading settings, just make it clear what the static front page & post page is set to.
Even if the blog functionality is not there right now, we want to accommodate
![screen shot 2018-03-22 at 3 33 27 pm](https://user-images.githubusercontent.com/5028555/37802191-a2beff82-2de6-11e8-9773-e7f93f639ccc.png)
 if they want it in the future.

**Comment by GPChappell** [comment]
Does this mean we need to create a static page to set the Post to even it doesn't currently exist? And style the same?

**Comment by rcass** [comment]
Yep exactly. I would just create a static page called 'Blog' or 'Journal' and yep style it the same.

Just be sure to check that index, single and archive pages look ok 👌
